pyccel/parser/syntactic.py

```diff
@@ -203,21 +182,7 @@ def parse(self, verbose=False):
 
     def _treat_iterable(self, stmt):
 
-        """
-        since redbaron puts the first comments after a block statement
-        inside the block, we need to remove them. this is in particular the
-        case when using openmp/openacc pragmas like #$ omp end loop
-        """
-
-        ls = [self._visit(i) for i in stmt]
-
-        if isinstance(stmt, (list, ListNode)):
-
-            return List(*ls, sympify=False)
-        elif isinstance(stmt, (tuple, TupleNode)):
-            return PythonTuple(*ls)
-        else:
-            return Tuple(*ls, sympify=False)
+        return [self._visit(i) for i in stmt]
 
     def _visit(self, stmt):
         """Creates AST from FST."""
```

Comment:
Should this docstring read "Create Sympy AST from Python FST?"

Reply:
I don't know. Maybe 'create pyccel ast from python ast'?

I'm not clear on the difference between ast and fst.
I'm not sure how many of the results are still stored in sympy objects. I think it might be a mixture of pyccel and sympy ast still